README.md

```diff
@@ -7,19 +7,22 @@ Yet another F-Droid client.
 
 ## Description
 
-Unofficial F-Droid client that resembles the classic F-Droid client.
+Unofficial F-Droid client resembling the classic F-Droid client.
 
-The client supports all basic F-Droid features, such as repository management and package installation. It doesn't
-support privileged extension, root installation, or local repositories.
+Jump over the lazy dog, manage repositories and install software quickly.
+No privileged extension, root installation, or sharing local repositories nearby yet.
```

Comment:
Didn't get the "Jump over the lazy dog" first, but it's actually funny :) Probably it's a good idea to rework it this way.

Reply:
The only problem here is "nearby yet", which implies these features may be available in the future. "local repositories" is something that will never be available in Foxy Droid. And I don't think it's right to leave assumptions that these features may be available in the future.